### ISSUE TITLE
[Feature request] Percent encoding support

### ISSUE THREAD

**Issue by Kristinita** [alvesjtiago/hover-preview#15]
### 1. Request

It would be nice, **if** users can preview images by links with [**percent encoding**](https://en.wikipedia.org/wiki/Percent-encoding).

### 2. Justification

I'm Russian man. I work with Russian-language images. Now I can't preview decoded **and** non-decoded images with Russian-language symbols.

### 3. Settings

**For** example, I have `test.md` file. It content:

```markdown
1. ![photo 1](https://upload.wikimedia.org/wikipedia/commons/thumb/c/c0/Cleopatras.needle.from.thames.london.arp.jpg/781px-Cleopatras.needle.from.thames.london.arp.jpg)
2. ![photo 2](https://upload.wikimedia.org/wikipedia/ru/thumb/d/db/%D0%9C%D0%BE%D0%BD%D1%83%D0%BC%D0%B5%D0%BD%D1%82_%D0%92%D0%BE%D0%B8%D0%BD%D1%81%D0%BA%D0%BE%D0%B9_%D0%A1%D0%BB%D0%B0%D0%B2%D1%8B_%28%D0%92%D0%B8%D0%B41%29.jpg/320px-%D0%9C%D0%BE%D0%BD%D1%83%D0%BC%D0%B5%D0%BD%D1%82_%D0%92%D0%BE%D0%B8%D0%BD%D1%81%D0%BA%D0%BE%D0%B9_%D0%A1%D0%BB%D0%B0%D0%B2%D1%8B_%28%D0%92%D0%B8%D0%B41%29.jpg)
3. ![photo 3](https://upload.wikimedia.org/wikipedia/ru/thumb/d/db/Монумент_Воинской_Славы_(Вид1).jpg/320px-Монумент_Воинской_Славы_(Вид1).jpg)
```

I successful open all photos.

### 4. Steps to reproduce

I hover my mouse to URL's of photo 1, 2, 3.

### 5. Expected behavior

**If** photo 1:

![Photo 1](http://i.imgur.com/LJLHdLQ.png)

### 6. Actual behavior

**If** photo 2, I can't preview photo. I get stack trace in Sublime Text console:

```python
Traceback (most recent call last):
  File "D:\Sublime Text 3 x64\sublime_plugin.py", line 389, in run_callback
    expr()
  File "D:\Sublime Text 3 x64\sublime_plugin.py", line 620, in <lambda>
    run_callback('on_hover', callback, lambda: callback.on_hover(v, point, hover_zone))
  File "hover_preview in D:\Sublime Text 3 x64\Data\Installed Packages\Hover Image Preview.sublime-package", line 100, in on_hover
  File "./python3.3/urllib/request.py", line 156, in urlopen
  File "./python3.3/urllib/request.py", line 475, in open
  File "./python3.3/urllib/request.py", line 587, in http_response
  File "./python3.3/urllib/request.py", line 513, in error
  File "./python3.3/urllib/request.py", line 447, in _call_chain
  File "./python3.3/urllib/request.py", line 595, in http_error_default
urllib.error.HTTPError: HTTP Error 404: Not Found
```

**If** photo 3, I don't get preview **and** stack trace.

Thanks.

**Comment by alvesjtiago** [comment]
@Kristinita the last version (0.0.12) that I just released fixes the problem with number 2. The problem with number 3 however has to do with the way the parentheses are being fetched so I'll have to change the way links are being fetched from the files which will take a longer. I'll let you know once I have a chance to fix that. Thank you

**Comment by MateuszDabrowski** [comment]
+1 for issue - just installed your plugin @alvesjtiago , but on all of my images I got just small, grey square pop up instead of image.

![zrzut ekranu 2017-06-11 o 12 09 38](https://user-images.githubusercontent.com/13339456/27010188-f51cd722-4e9e-11e7-8e45-b53af58f8376.png)


URLs: 
```
1. <img src="http://images.go.wolterskluwer.com/EloquaImages/clients/WoltersKluwer/%7B2f6dbcd1-eeba-4888-af4a-f4b26d0c6574%7D_logo.png" class="header__logo-img" />
2. background: url('http://secure.p06.eloqua.com/EloquaImages/clients/WoltersKluwer/%7B9584504f-22b7-4c22-a3e2-f4668c53f9b7%7D_cta-arrow.png') no-repeat right center #e5202e !important;
3. <a href="#pobieranie"><img class="hidden_desktop" src="http://images.go.wolterskluwer.com/EloquaImages/clients/WoltersKluwer/%7B7e35fe03-8d95-4e6f-8544-d48ba64d3a67%7D_HR_Index_550x600.png"></a>
```

Neither works, however I can preview both 1. and 2. from original issue by @Kristinita

**Comment by alvesjtiago** [comment]
Thank you for reporting the issue @MateuszDabrowski. I've just released version 0.0.14 which fixes what you have reported. Please let me know if it is now working for you. I'll get back to the remaining parentheses issue reported by @Kristinita once I have the chance.

**Comment by MateuszDabrowski** [comment]
@alvesjtiago Fixed :) Thank you!

**Comment by AmjadHD** [comment]
#36 implements highlight + hover as a workaround for number 3 and the like

**Comment by AmjadHD** [comment]
#37 fixes number 3

**Comment by alvesjtiago** [comment]
This is addressed with #37 and version 0.1.0 which should be automatically updated via package manager.
